File: c/136_pattern_28.c
#include<stdio.h>
int min(int a, int b) {
    return a < b ? a : b;
}
int main() {
    int n;
    printf("Enter n :- ");
    scanf("%d",&n);
    for(int i=0;i<2*n;i++) {
        for(int k=0;k<=min(i,2*n-i-1);k++) {
            printf("*");
        }
        printf("\n");
    }
}

/*

Output :-
Enter n :- 5
*
**
***
****
*****
*****
****
***
**
*

*/

File: python/code/136_pattern28.py
n = int(input("Enter n :- "))
for i in range(2*n):
    for k in range(1+min(i,2*n-i-1)):
        print("*", end='')
    print()
    
# Output :-
# Enter n :- 5
# *
# **
# ***
# ****
# *****
# *****
# ****
# ***
# **
# *